ld_script_modern.ld

```diff
@@ -15,26 +15,20 @@ SECTIONS {
     ewram 0x2000000 (NOLOAD) :
     ALIGN(4)
     {
-        src/*.o(ewram_data);
-        gflib/*.o(ewram_data);
+        gflib/malloc.o(ewram_data);
+        src/decompress.o(ewram_data);
+        . = 0x20000;
```

Comment:
Is this necessary? If we just had `*.o(ewram_data);` would anything break as a result of the heap/decompression buffer having moved?

Reply:
It has to be word aligned for starters so this helps